bindings/flow/fdb_flow.actor.cpp

```diff
@@ -187,51 +262,47 @@ namespace FDB {
 		return fdb_error_predicate( pred, e.code() );
 	}
 
-	Reference<Cluster> API::createCluster( std::string const& connFilename ) {
-		return Reference<Cluster>(new Cluster(connFilename));
-	}
-
-	Reference<DatabaseContext> API::createDatabase(std::string const& connFilename) {
-		FDBDatabase *db;
+    Reference<Database> API::createDatabase(std::string const& connFilename) {
+	    FDBDatabase *db;
 		throw_on_error(fdb_create_database(connFilename.c_str(), &db));
-		return Reference<DatabaseContext>(new DatabaseContext(db));
-	}
+	    return Reference<Database>(new DatabaseImpl(db));
```

Comment:
Mixed spaces and tabs here and in a bunch of following lines. I won't mark every one.

Reply:
I just ran clang-format with `git-clang-format` and assumed it would put it in the format, project suggested. Let me check.